tests/integration/link/test_link_children.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import dash
+from dash.dependencies import Input, Output
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+@pytest.mark.DCC776
+def test_children001_default(dash_dcc):
```

Comment:
the convention we've been using elsewhere (though I see it's only used sporadically in this repo, but let's try to move in this direction) for the test id is `ccffnnn`
- `cc` is 2 chars for the component (`li`?)
- `ff` is 2 chars for the file (`ch`?)
- `nnn` is a serial number within the file

```suggestion
def test_lich001_default(dash_dcc):
```
That way you can run all tests in this file with `pytest -k lich` or just this single test `pytest -k lich001` etc and you can be pretty sure these target just the tests you want.